src/components/Contacts/ContactsLeftPanel/ContactsLeftPanel.test.tsx

```diff
@@ -60,32 +63,50 @@ const mocks = {
   },
 };
 
+const mapData = coordinatesFromContacts(
+  {
+    nodes: [...mocks.Contacts.contacts.nodes],
+    pageInfo: { endCursor: 'Mg', hasNextPage: false },
+    totalCount: 1,
+  },
+  'en-US',
+);
+
 describe('ContactsLeftPanel', () => {
   it('pans to the selected contact', async () => {
     const panTo = jest.fn();
     const setZoom = jest.fn();
-    const Component = () => {
-      const { mapRef } = useContext(ContactsContext) as ContactsType;
-
-      useEffect(() => {
-        mapRef.current = {
-          panTo,
-          setZoom,
-        } as unknown as google.maps.Map;
-      }, []);
-
-      return null;
-    };
 
     const { findByText } = render(
       <ThemeProvider theme={theme}>
         <TestRouter router={router}>
           <GqlMockedProvider<{ UserOption: UserOptionQuery }> mocks={mocks}>
             <ContactsWrapper>
-              <>
-                <Component />
+              <ContactsContext.Provider
```

Comment:
Do you know why this helps? `ContactsWrapper` already wraps its children in `<ContactsContext.Provider>`.

Reply:
My guess is that the `viewMode` was getting set randomly, but I could be wrong. I figured if I controlled all the properties, I would know for sure it would pass.